src/common/entities/students.entity.ts

```diff
@@ -1,34 +1,34 @@
-import { Entity, PrimaryKey, Property, OneToOne, Enum } from '@mikro-orm/core';
+import { Entity, PrimaryKey, Property, OneToOne, Enum } from "@mikro-orm/core";
 
-import { EEducationLevel, EMedium } from '../enums/students.enums';
-import { CustomBaseEntity } from './custom-base.entity';
-import { User } from './users.entity';
+import { EEducationLevel, EMedium } from "../enums/students.enums";
+import { CustomBaseEntity } from "./custom-base.entity";
+import { User } from "./users.entity";
 
-@Entity({ tableName: 'students' })
+@Entity({ tableName: "students" })
```

Comment:
Each entity should have a designated repository.
Please look into the template for how to do this.